solr/solrj/src/java/org/apache/solr/common/cloud/ConnectionManager.java

```diff
@@ -155,27 +155,28 @@ public void process(WatchedEvent event) {
                 public void update(SolrZooKeeper keeper) {
                   try {
                     waitForConnected(Long.MAX_VALUE);
+                    
+                    log.info("Connection with ZooKeeper reestablished.");
```

Comment:
we already log this as debug in `waitForConnected`. Let's just switch that out to an info ?